modules/test-tools/src/main/kotlin/jp/co/soramitsu/iroha2/testengine/IrohaConfig.kt

```diff
@@ -24,17 +24,15 @@ import java.util.function.Consumer
 class IrohaConfig(
     var networkToJoin: Network = newNetwork(),
     var logConsumer: Consumer<OutputFrame> = Slf4jLogConsumer(getLogger(IrohaContainer::class.java)),
-    var genesis: Genesis = Genesis.getEmpty(),
+    var genesisPath: String? = null, // first option
+    var genesis: Genesis? = Genesis.getEmpty(), // second option
```

Comment:
why nullable if default value is going to always be here

Reply:
then why not `null` as a default value and throw an exception if nothing is set, or there is a point in running Iroha with empty genesis?